ui/components/app/advanced-gas-controls/index.scss

```diff
@@ -0,0 +1,36 @@
+.advanced-gas-controls {
+  &__row {
+    margin-bottom: 20px;
+  }
+
+  &__row-heading {
+    display: flex;
+  }
+
+  &__row-heading-tooltip,
+  /* TODO: Why doesn't `typography--display-inline-block` do its job?! */
+  &__row-heading h6 {
+    display: inline-block;
+  }
+
+  &__row-heading-detail {
+    flex-grow: 1;
+    align-self: center;
+  }
+
+  &__row-error,
+  &__row--error h6 {
+    color: $error-1 !important;
+    padding-top: 6px;
+  }
+
+  h6 {
+    padding-bottom: 6px;
+    margin-inline-end: 6px;
+  }
+
+  i {
+    color: #dadada;
+    font-size: 0.75rem;
```

Comment:
there are variables for accessing these values without all the other typography settings. $font-size-h7 in this case

Reply:
Good shout!